packages/@okta/vuepress-site/docs/guides/registration-inline-hook/send-response-code/nodejs/send-response.md

```diff
@@ -0,0 +1,51 @@
+For this example, only emails containing "yourDomain.com" are allowed and can self-register. You can also include an `error` object in the response for those users denied registration.
+
+See the [response properties](/docs/reference/registration-hook/#objects-in-the-response-from-okta) of a Registration Inline Hook for full details.
+
+
+```javascript
+app.post('/registrationHook', async (request, response) => {
+  console.log(" "); // for separation of logs during testing
+  console.log('Email for ' + request.body.data.userProfile['firstName'] + " " + request.body.data.userProfile['lastName'] + " " + request.body.data.userProfile['email']);
+
+  var emailRegistration = request.body.data.userProfile['email'];
+  if (emailRegistration.includes("yourDomain.com")){
```

Comment:
```suggestion
  if (emailRegistration.includes("example.com")){
```

Reply:
Please ensure that all of the example domains in this doc are `example.com`.

Reply:
Yes, did a search and replace. And will do so in Glitch projects as well.